raytracing.py

```diff
@@ -4,6 +4,11 @@
 w = 400
 h = 300
 
+# Calculating the intersection of the ray, starting from some point of the object's surface, with other objects,
+# we need to offset the ray's start point by normal from the surface.
+# Otherwise the ray may intersect current object in it's starting point.
+surface_offset = .0001
```

Comment:
Constants according to a PEP8 should be capitalized

Reply:
Fixed